app/src/main/java/org/mozilla/focus/popup/PopupUtils.kt

```diff
@@ -65,11 +67,18 @@ object PopupUtils {
     }
 
     private fun setOrigin(hostInfo: TextView, origin: String?, url: String) {
+        val LOGTAG = "PopupUtils"
```

Comment:
This should be a constant at the top of the class/object:

```Kotlin
const val LOGTAG = "PopupUtils"
```

Or let's just not log this at all... :)